### PR TITLE
Add RFCs re Test Directory Naming Unification and Router Service Refresh Method

### PR DESCRIPTION
**Pull request by jaredgalanis**

### CODE REVIEW

File: source/2020-05-29-the-ember-times-issue-150.md [hunk 1]
```diff
@@ -57,25 +59,25 @@ What's in store for the future? The [project](https://github.com/ember-learn/emb
...
+## [New Router Service Refresh RFC ♻️](https://github.com/emberjs/rfcs/pull/631)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There's a cool 😎 new RFC by [Gaurav Munjal (@Gaurav0)](https://github.com/Gaurav0) regarding adding a `refresh` method to the router service.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Note this `refresh` method would be different than the currently available `Route.prototype.refresh` method that refreshes only the refreshes whatever route you call it on and all routes “below” it in the route hierarchy. This newly proposesd method would refresh all routes.
```
Comment (ijlee2) [inline source/2020-05-29-the-ember-times-issue-150.md]: I'm not familiar with the RFC so I wasn't sure what you meant by "only the refreshes whatever route ... in the route hierarchy." Is it possible to rephrase this part of the sentence? (If not, no worries.)

Can we fix the spelling from `proposesd` to `proposed`?

**Review by ijlee2 — APPROVED** [review]
LGTM overall! Thanks for covering both RFCs!

File: source/2020-05-29-the-ember-times-issue-150.md (hunk 1 above)
Comment (amyrlam) [reply source/2020-05-29-the-ember-times-issue-150.md]: thanks, taking a looksie at the RFC re: your comment  👀

File: source/2020-05-29-the-ember-times-issue-150.md [hunk 2]
```diff
@@ -57,25 +59,25 @@ What's in store for the future? The [project](https://github.com/ember-learn/emb
...
+## [New Router Service Refresh RFC ♻️](https://github.com/emberjs/rfcs/pull/631)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There's a cool 😎 new RFC by [Gaurav Munjal (@Gaurav0)](https://github.com/Gaurav0) regarding adding a `refresh` method to the router service.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Note this `refresh` method would be different than the currently available `Route.prototype.refresh` method that refreshes only the refreshes whatever route you call it on and all routes “below” it in the route hierarchy. This newly proposed method would refresh all routes.
```
Comment (amyrlam) [inline source/2020-05-29-the-ember-times-issue-150.md]:
```suggestion
Note: This RFC is suggesting a `refresh` method that would be different than the currently available `Route.prototype.refresh` method, which refreshes whatever route you call it on and all routes “below” it in the route hierarchy. 

This newly proposed method would refresh all currently active routes.
```

File: source/2020-05-29-the-ember-times-issue-150.md (hunk 1 above)
Comment (jaredgalanis) [reply source/2020-05-29-the-ember-times-issue-150.md]: Thanks for getting the adjustments worked out @amyrlam and @ijlee2 !

Comment (amyrlam) [reply source/2020-05-29-the-ember-times-issue-150.md]: 👍 really appreciate the writeups, thank you!

**Pull request merged**